o8g/scripts/actions.py

```diff
@@ -362,6 +363,12 @@ def has_deck(card, x=0, y=0):
 def discard(card, x=0, y=0):
   pile = get_discard_pile(card)
   if pile is not None:
+    if card.markers[HONORED]:
+      me.honor+=1
+      notify("{} gains 1 honor from {} honored status.".format(me, card))
```

Comment:
I wanted to give a bit more information about where the gain or loss of honor was coming from.
Doesn't really matter when you discard only one card, but when you select and discard many, like you would in fate phase, it can be usefull.
But maybe it's too much yeah